kpatch-build/kpatch-build

```diff
@@ -649,6 +658,26 @@ module_name_string() {
 	echo "${1//[^a-zA-Z0-9_-]/-}" | cut -c 1-55
 }
 
+is_supported_distro(){
+    distro=$1
+    for each_distro in ${SUPPORTED_DISTRO[@]}; do
+        if [[ $distro == $each_distro ]]; then
+            return 1;
+        fi
+    done;
```

Comment:
Unnecessary semicolon (here and elsewhere)

Reply:
OK, in my new commit, I deleted this unnecessary semicolon~~